### PR TITLE
Extend documentation, add documentation on how to change the CSR data model

### PR DESCRIPTION
**Pull request by ewelinagr**

### CODE REVIEW

File: csr2transmart/mappers/observation_mapper.py [hunk 1]
```diff
@@ -140,6 +146,14 @@ def map_observation_metadata(self, entity_type_to_id: Dict[str, str]) -> Optiona
 
     def get_observation_for_value(self, row_value, concept: Concept, metadata: ObservationMetadata,
                                   patient: Patient) -> Observation:
+        """
+        Get trasmart-loader Observation
```
Comment (gijskant) [inline csr2transmart/mappers/observation_mapper.py]: Typo: `trasmart`, copy-pasted below

File: changing-data-model.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+For more details see `official pydantic documentation`_.
+
+.. _`official pydantic documentation`: https://pydantic-docs.helpmanual.io/
+
+New entities has to be added to `<csr/subject_registry_writer.py>`_ and `<csr/subject_registry_reader.py>`_.
+
+Changes in `csr2transmart` package:
+-----------------------------------
+
+In order to include observations for the new entity when mapping input data to the TranSMART data model,
```
Comment (gijskant) [inline changing-data-model.rst]: This part is not required anymore, right?

File: changing-data-model.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+-------------------------
+
+The data model is defined as a data class in `<csr/csr.py>`_.
+In order to add a new entity, a new class has to be defined.
+
+Classes are defined using `pydantic` - library to validate JSON documents and convert them into Python objects, 
+based on Python Type Annotations.
+
+All new entities have to have a non-nullable `identity` defined. They also have to be a part of `SubjectEntity` union 
+and `CentralSubjectRegistry` class defined in `csr.py`.
```
Comment (gijskant) [inline changing-data-model.rst]: Not the `CentralSubjectRegistry` anymore.

File: changing-data-model.rst (hunk 2 above)
Comment (ewelinagr) [reply changing-data-model.rst]: oh no, I lost one commit during rebasing... I will remove this part.

**Pull request merged**